src/data/brk.prepare.json

```diff
@@ -195,7 +195,34 @@
     },
     {
       "type": "import_csv",
-      "source": "http://www.kadaster.nl/schemas/waardelijsten/BurgemKadgemSectie/BurgemKadgemSectie.csv",
+      "source": "https://developer.kadaster.nl/schemas/waardelijsten/AardAantekening/AardAantekening.csv",
```

Comment:
This changes the datasource to kadaster's developer environment. Is this intentional?